modules/distribution-zones/src/main/java/org/apache/ignite/internal/distributionzones/DistributionZoneManager.java

```diff
@@ -947,7 +971,8 @@ private void initDataNodesAndTriggerKeysInMetaStorage(
                             revision
                     );
                 } else if (res.getAsBoolean()) {
-                    LOG.debug("Update zones' dataNodes value [zoneId = {}, dataNodes = {}, revision = {}]", zoneId, dataNodes, revision);
+                    LOG.debug("Update zones' dataNodes value [zoneId = {}, dataNodes = {}, revision = {}]",
```

Comment:
I think this should be on info level.

Reply:
I don't want to change it in my pr. So I have just reverted logging changes.

Reply:
I changed a log level to info in several places where a data nodes value is updated.